benchData/time.csv

```diff
@@ -0,0 +1,21 @@
+Section,Start Time,End Time,Time Elapsed
+Create Tasks,0.01173190000008617,0.011608300001171301,-0.0001235999989148695

```

Comment:
What is section? @kylevitale2

Reply:
Section is just a label for us to determine what section of code we were timing